### PR TITLE
feat: Add staking::{chill, validate} and session::setKeys

### LINKED ISSUE #32: Add new functions
Add:

- `staking::validate`
- `staking::chill`
- `session::set_keys`

### PR DESCRIPTION
**Pull request by amaury1093**
fixes #32

### CODE REVIEW

File: src/methods/staking/chill.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { BaseTxInfo, createMethod, UnsignedTransaction } from '../../util';
+
+/**
+ *Declare no desire to either validate or nominate.
```
Comment (joepetrowski) [inline src/methods/staking/chill.ts]:
```suggestion
 * Declare the desire to cease validating or nominating. Does not unbond funds.
```

File: src/methods/staking/validate.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import {
+  Args,
+  BaseTxInfo,
+  createMethod,
+  UnsignedTransaction
+} from '../../util';
+
+export interface StakingValidateArgs extends Args {
+  /**
+   * Declare the desire to nominate `targets` for the origin controller.
```
Comment (joepetrowski) [inline src/methods/staking/validate.ts]: In the rust code, this is a `Perbill`. Does submitting e.g. 5 here imply a 5% commission?
```suggestion
   * Set the desired commission for the validator. Value is {percent, Perbill, ???}.
```

**Review by joepetrowski — COMMENTED** [review]
Just a few doc changes. Also, need to run `yarn docs`.

File: src/methods/staking/validate.ts (hunk 2 above)
Comment (amaury1093) [reply src/methods/staking/validate.ts]: No it's perbill here too.

**Pull request merged**